src/features/instance/applications/index.tsx

```diff
@@ -7,19 +7,13 @@ export function ApplicationsIndex() {
 		<div className="flex flex-col justify-center gap-4 min-h-[calc(80vh-theme(spacing.20))]">
 			<h1 className="text-2xl font-bold text-center text-white">Applications</h1>
 			<div className="flex flex-col items-center justify-center gap-8 md:flex-row">
-				<Link
-					to={`editor`}
-					className="w-full max-w-80"
-				>
+				<Link to="editor" className="w-full max-w-80">
 					<Button className="w-full py-10 text-lg" variant="positiveOutline">
 						<Edit />
 						Edit Applications
 					</Button>
 				</Link>
-				<Link
-					to={`new`}
-					className="w-full max-w-80"
-				>
+				<Link to="new" className="w-full max-w-80">
```

Comment:
These changes weren't necessary, but I thought it would be nice to simplify them ever so slightly, and have them for reference when considering the change to the EmptyApplicationsView. 🥳